backend/src/packages/chaiNNer_standard/utility/value/formats.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
+
+from enum import Enum
+
+from nodes.properties.inputs import EnumInput
+from nodes.properties.outputs import NumberOutput
+
+from .. import value_group
+
+
+class ResList(Enum):
+    HD_1080 = "HD_1080"
+    NTSC_MPEG2 = "NTSC MPEG2"
+    NTSC_DV = "NTSC_DV"
+    NTSC = "NTSC"
+    PAL = "PAL"
+    PAL_MPEG2 = "PAL MPEG2"
+    HD_720 = "HD_720"
+    UHD_4K = "UHD_4K"
+    UHD2_8K = "UHD2_8K"
+    DCI_2KFLAT = "DCI_2KFLAT"
+    DCI_2KSCOPE = "DCI_2KSCOPE"
+    DCI_2KFULL = "DCI_2K1080"
+    DCI_4KFLAT = "DCI_4KFLAT"
+    DCI_4KSCOPE = "DCI_4KSCOPE"
+    DCI_4KFULL = "DCI_4K2160"
+    SQ256 = "SQ256"
+    SQ512 = "SQ512"
+    SQ1024 = "SQ1024"
+    SQ2048 = "SQ2048"
+    SQ4096 = "SQ4096"
+    SQ8192 = "SQ8192"
+
+
+@value_group.register(
+    schema_id="chainner:utility:formats",
+    name="Formats",
+    description="Industry formats.",
```

Comment:
Let's explain the node a bit more.

```suggestion
    description="Returns the image dimensions of industry formats. E.g. HD 1080p and UHD 4K.",
```